src/client/utils/theme/index.ts

```diff
@@ -1,8 +1,8 @@
-import light from './themes/light';
 import { UserTheme } from '../../store/theme/types';
 import { Theme } from './type';
+import dark from './themes/dark';
 
-export default function changeTheme(theme:UserTheme = light) {
+export default function changeTheme(theme:UserTheme = dark) {
```

Comment:
```suggestion
export default function changeTheme(theme: UserTheme = dark) {
```
eslint
И разве на клиенте у нас должны быть темы? может логичней предзагружать тему из базы?